src/main/java/net/datafaker/providers/base/Pronouns.java

```diff
@@ -0,0 +1,44 @@
+package net.datafaker.providers.base;
+
+/**
+ * This class generates gender neutral pronouns
+ * @since 2.1.0
+ */
+public class Pronouns extends AbstractProvider<BaseProviders> {
```

Comment:
Any reason why a new class is used, and not add it to Gender? Gender seems like a reasonable place for this, or not?

Reply:
That's a good question.
I'm trying to follow the same pattern as the other " self-contained " providers. They have not implemented the fluent interface approach as I could call the class itself or another class.

Example:
```java
faker.gender().pronouns().subjective();
```

The example above would be implemented in this way:
* The `Pronouns` class still exists
* The `Gender` class would have a pronouns() method returning its new instance

```java
public class Gender extends AbstractProvider<BaseProviders> {

    // ignoring already existing code
    public Pronouns pronouns() {
        return new Pronouns(faker);
    }
}
```
* The `Pronouns` class as a default `toString()` in the case the consumer would call only its method
```java
faker.gender().pronouns()
```

I'm open to any implementation ideas :-)

Reply:
@bodiam any feedback about this last comment?

Reply:
same could achieved with 
```java
    public Pronouns pronouns() {
        return faker.getPronouns();
    }
```
however no need to make GC happy by creation a new object each time there is a call to `pronouns`

Reply:
@eliasnogueira it's a bit comment, not sure what to say :-) I'm not a fan of the fluent interface approach that much, so if you want to keep the Pronouns as a separate class, that's fine.